css-two-column-layout/styles.css

```diff
@@ -0,0 +1,73 @@
+.container {
+  width: 1000px;
+  margin-right: auto;
+  margin-left: auto;
+  font-family: 'Karla', sans-serif;
+  font-style: normal;
+}
+
+h1 {
+  font-weight: 700;
+  text-transform: uppercase;
+  font-size: 48px;
+  margin-bottom: 20px;
+}
+
+h2 {
+  font-style: normal;
+  font-weight: 700;
+  text-transform: uppercase;
+  font-size: 16px;
+}
+
+p {
+  font-style: normal;
+  font-weight: 400;
+  line-height: 1.8;
+  font-size: 14px;
+  padding-right: 8px;
+  padding-left: 4px;
+}
+
+.header-para {
+  padding-bottom: 30px;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.column-full {
+  width: 100%;
+  text-align: center;
+  padding-right: 200px;
+  padding-left: 200px;
```

Comment:
Instead of using such a large value for padding, I recommend setting a width on this element, then centering with `margin: auto`

Reply:
I think is left-over code from something that got edited later...can I just remove, instead?